topics/prometheus/README.md

```diff
@@ -3,7 +3,7 @@
 
 
 # Prometheus Architecture
-<img src="https://prometheus.io/assets/architecture.png" width="700">
+<img src="https://prometheus.io/assets/images/prometheus/prometheus-architecture.png" width="700">
```

Comment:
Thank you @MohitKambli for your contribution, just a minor comment at this line. It should be the related path to the assets folder, something like: src="../../assets/images/prometheus/prometheus-architecture.png".
Can you please check and update this path? Thanks